deno/import_map.json

```diff
@@ -1,5 +1,7 @@
 {
   "imports": {
+    "$denoRoot/": "./",
```

Comment:
I figured it would be clearer to differentiate between the root of the deno project and of the entire project